packages/streamsets/streamsets.patch

```diff
@@ -0,0 +1,26 @@
+diff -x '*.tgz' -x '*.lock' -uNr packages/streamsets/charts-original/Chart.yaml packages/streamsets/charts/Chart.yaml
+--- packages/streamsets/charts-original/Chart.yaml
++++ packages/streamsets/charts/Chart.yaml
+@@ -2,15 +2,18 @@
+ appVersion: 3.8.0
+ description: Control Agent for managing StreamSets Control Hub Deployments
+ home: https://streamsets.com
+-icon: https://streamsets.com/wp-content/uploads/2016/08/StreamSets-logo-small.png
++icon: https://github.com/streamsets/datacollector/raw/master/basic-lib/src/main/resources/sdcipc.png
+ keywords:
+ - streamsets
+ - sdc
+ - sch
+ maintainers:
+-- email: adam@streamsets.com
+-  name: kunickiaj
++- email: thomas.ganka@streamsets.com
++  name: thomasganka
+ name: control-agent
+ sources:
+ - https://github.com/streamsets/helm-charts/tree/master/incubating/control-agent
+ version: 2.0.1
++annotations:
++  catalog.cattle.io/certified: partner
++  catalog.cattle.io/release-name: streamsets-v1.0.0
```

Comment:
Please revert this to `streamsets`. Rancher uses this for deployment and having `.` in it will cause the installation to fail. The UI displays the chart's version based on its metadata. 

```
helm  install --namespace=default --timeout=10m0s  --values=/home/shell/helm/values-control-agent-2.0.100.yaml  --version=2.0.100 --wait=true streamsets-v1.0.0  /home/shell/helm/control-agent-2.0.100.tgz

rror: Deployment.apps "streamsets-v1.0.0-control-agent" is invalid: spec.template.spec.containers[0].name: Invalid value: "streamsets-v1.0.0-control-agent": a DNS-1123 label must consist of lower case alphanumeric characters or '-', and must start and end with an alphanumeric character (e.g. 'my-name', or '123-abc', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?')
```